blocks/sticky-footer/sticky-footer.js

```diff
@@ -97,6 +104,9 @@ export default async function decorate(block) {
   mobileNumberField.addEventListener('input', handleMobileNumberValidation);
   stickyFooterContainer.appendChild(mobileNumberField);
 
+  const captchaContainer = createReCaptcha();
+  stickyFooterContainer.appendChild(captchaContainer);
```

Comment:
@vivesing recaptch appears with a little delay, any probable reason for the same ?

Reply:
Yes its intended beacuse, recapcha is injeced everywhere into the page in a delayed phase. see here
https://github.com/aemsites/icicidirect/blob/b420664fd055d8deaf2c923a52db8c8e8ba173fc/scripts/delayed.js#L15